### PR TITLE
To Do List structure

### PR DESCRIPTION
**Pull request by maaloum**
### In this PR I:
-  Set up a new project with webpack that is based on the webpack exercise you have already completed.
- Created an index.html file.
- The index.html file are set as a template using the HTML Webpack Plugin.
- Created an index.js file and set an array of some simple to do tasks (array of objects). Each task object should contain three keys:
        description [string].
        completed [bool].
        index: [number].
- Wrote a function to iterate over the tasks array and populate an HTML list item element for each task.
- On page load render the dynamically created list of tasks in the dedicated placeholder. The list should appear in order of the index values for each task.
- Create a style.css and set rules for the To Do List. CSS must be loaded by Webpack Style/CSS Loader.

### CODE REVIEW

**Review by DammyShittu — COMMENTED** [review]
Hi @maaloum,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

### You have submitted a code review request that has failed linters

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+}
+
+.tasks svg {
+  width: 20px;
+  height: 20px;
+}
+
+/* .hide {
+  display: none;
+} */
```
Comment (elmar8287) [inline src/style.css]: - [x] Please, consider removing not using code - it will make your code professional 👍

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
+// import './style.css';
+import './style.css';
+
+// creating tasks
+const tasks = [{
+  description: 'Gym',
+  completed: true,
+  index: 0,
+}, {
+  description: 'Code',
```
Comment (elmar8287) [inline src/index.js]: - [ ] Please, consider removing the comments to make the code clear - your code is understandable and talks about what happening 👍

**Review by elmar8287 — CHANGES_REQUESTED** [review]
Hi, @maaloum.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Very good PR description 🥇 
- Nice README file 👍 

## Required Changes ♻️

- [ ] Check the comments under the review.

## Optional suggestions

- N.A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
### Please, remember to tag me (@elmar8287) in your question so I can receive the notification 👍 
giphy

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AmaduKamara — APPROVED** [review]
# Status: Approved 🟢🟢🟢 

Hi @maaloum,

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉 

## Highlights:

✅ You used Gitflow correctly
✅ No linter error
✅ Good PR description
✅ App is setup with Webpack correctly 👍

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**